tachyon/math/finite_fields/generator/generator_util.cc

```diff
@@ -110,7 +110,7 @@ std::string GenerateInitField(std::string_view name, std::string_view type,
   if (base::ConsumePrefix(&value, "-")) {
     ss << "-";
   }
-  ss << type << "::FromDecString(\"" << value << "\");";
+  ss << type << "::FromDecString(\"" << value << "\").value();";
```

Comment:
It's guranteed, since we input from bazel build file. Also i don't know why but if i do this, it wont be compiled on ubuntu!

Reply:
In addition, `value()` is the one that crashes the program and if you access using `*`, it doesn't.

Reply:
> It's guranteed, since we input from bazel build file. Also i don't know why but if i do this, it wont be compiled on ubuntu!

1. I accidentally put an extra quotation mark (") in the suggestion -> I tried running it and it seems to work fine?
2. Right, `value()` would crash the program, but `*` would not, which is why I suggested this fix, but if `value` is guaranteed to not be malformed, there's no need to change, I suppose.

Reply:
Oh, if I can build with your suggestion, then it would be better with `*`! Let me try!